scala-mode2-prettify.el

```diff
@@ -0,0 +1,54 @@
+;;; scala-mode-prettify.el - Extension of scala-mode for prettifying scala symbols/code
+;;; Copyright (c) 2016 Merlin Göttlinger
+;;; For information on the License, see the LICENSE file
+
+(defconst scala-mode-pretty-bool-alist '(
```

Comment:
it's pretty unusual to use `defconst`. It might be better to use `defcustom` (which allows the user to customise) and also provide a documentation string.